.gitignore

```diff
@@ -7,3 +7,4 @@ jest.json
 log.txt
 node_modules
 npm-debug.log
+.vscode/settings.json
```

Comment:
Common practice here is to have such IDE specific ignores in the local gitignore, not project's one (please change)